content/06.model-complexity.md

```diff
@@ -1,36 +1,35 @@
-### Reducing misinterpretation of model output with statistical techniques 
+## Reducing misinterpretation of model output with statistical techniques 
 
 
 Machine learning methods are generally accompanied by a few critical assumptions. 
-First, ML methods often work best on datasets that contain equal number of samples for all categories (no "class imbalance").
-Second, the dataset is complete; all samples have measurements for all variables in the dataset (i.e., the dataset is not "sparse", meaning that it is not missing data for some samples).
+First, the training dataset contains equal number of samples for all categories (no "class imbalance").
+Second, the training dataset is complete; all samples have measurements for all variables in the dataset (i.e., the dataset is not "sparse", meaning that it is not missing data for some of the samples).
 Third, there is no ambiguity about the labels for the samples in the dataset (i.e. no "label-noise"). 
 
 Rare disease datasets, however, violate many of these assumptions.
 There is generally a high _class imbalance_ due to small number of samples for specific classes (e.g., only a few patients with a particular rare disease in a health records dataset), the data are often _sparse,_ and there may be abundant _label-noise_ due to incomplete understanding of the disease.
 All of these contribute to low signal to noise ratio in rare disease datasets.
-Thus, applying ML to rare disease data without any accommodations for the above shortcomings may lead to models that lack stability required for reproducibility or simplicity required for interpretation.
-In this section, we highlight a few ML techniques that can help manage the challenges in applying ML models applied to rare disease data.
+Thus, applying ML to rare disease data without any accommodations for the above shortcomings may lead to models that have low reproducibility or are hard to interpret.
 
-Class imbalance in datasets can be addressed using decision tree-based ensemble learning methods (Figure[@fig:2]A-B) to increase the stability of ML predictions.
+Class imbalance in datasets can be addressed using decision tree-based ensemble learning methods (e.g. random forests) (Figure[@fig:3]a).
```

Comment:
Acknowledging that I'm favoring the American English usage of a comma after i.e. and e.g.

```suggestion
Class imbalance in datasets can be addressed using decision tree-based ensemble learning methods (e.g., random forests) (Figure[@fig:3]a).
```

I am highly attuned to this for whatever reason (my schooling, my schooling is the reason), so I'll probably be consistent in these edits/suggestions and consistency seems considerably more important than whether or not the comma is included.

Definitely not a hill I would die on.